src/libraries/System.Net.WebSockets.Client/src/System/Net/WebSockets/BrowserWebSockets/ReceivePayload.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices.JavaScript;
+
+namespace System.Net.WebSockets
+{
+    internal sealed class ReceivePayload
```

Comment:
Where is this type used?

Reply:
Used by BrowserWebSocket.cs to handle the received message queue